cmd/dhcpserver/get.go

```diff
@@ -47,12 +47,22 @@ var getCmd = &cobra.Command{
 			return err
 		}
 
-		servers, err := pvmclient.DHCPClient.Get(id)
+		server, err := pvmclient.DHCPClient.Get(id)
 		if err != nil {
 			return fmt.Errorf("failed to get a dhcpserver, err: %v", err)
 		}
-
-		spew.Dump(servers)
+		if server.Network.Name == nil {
+			klog.Infof("DHCP client reports status: %s, please retry in sometime.", *server.Status)
+			return nil
+		}
+		var IPandMAC string
+		for _, lease := range server.Leases {
+			IPandMAC += fmt.Sprintf("%s - %s ", *lease.InstanceIP, *lease.InstanceMacAddress)
```

Comment:
space is the delim here for the entries, wondering if there is an option of using newline and see how it looks?

Reply:
The closest i could bring by having a new line, introducing a space around `-` seems to push content the MAC address to the next line. Hope this is ok!
```
Kishens-MacBook-Pro  bin : pretty-dhcp  : ./pvsadm dhcpserver get --instance-id eeabbe9d-2dcd-4e04-aed3-eb35739cf0d2 --id 73bbe812-f758-4707-8170-2427f79eab0e
I0423 12:16:05.275645   78108 root.go:50] Using an API key from IBMCLOUD_API_KEY environment variable
+----------------------------------------------------+--------------------------------+--------+
|                    NETWORK NAME                    |            IP - MAC            | STATUS |
+----------------------------------------------------+--------------------------------+--------+
| DHCPSERVER6e9d7e95424d42ed8b7ff68305b03543_Private | 192.168.0.10-fa:16:3e:2f:66:e4 | ACTIVE |
|                                                    | 192.168.0.11-fa:16:3e:16:6f:02 |        |
|                                                    |                                |        |
+----------------------------------------------------+--------------------------------+--------+
```